app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
It was that which I gave a response above:
"The naming convention doesn't work exactly like that, I thought the suffix was meant to be the type for the class, not for replacing camel case?"

The file names should actuall be kebab case I think. A great name :D 

Does it say somewhere to name the files different from classes?